examples/seismic/acoustic/accuracy.ipynb

```diff
@@ -418,9 +515,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 18,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Convergence rate in time is: 1.8546\n"
+     ]
+    }
+   ],
    "source": [
```

Comment:
#NBVAL_IGNORE_OUTPUT

Reply:
How about %.3e, does really need to be %.2e?

Reply:
done